ci/citest/acceptance-test/multibox/ind-steps/step-qemu-kvm/postconfigure.sh

```diff
@@ -0,0 +1,27 @@
+#!/bin/bash
+
+procc_type=
+case ${ACCEL_TYPE} in
+    vmx) procc_type="intel" ;;
+    svm) procc_type="amd" ;;
+    *) echo "Invalid hardware acceleration" ; exit 255 ;;
```

Comment:
I would suggest using a different exit code than 255 because 255 has the special meaning: "exit code out of range." http://tldp.org/LDP/abs/html/exitcodes.html

It's not a big deal and can be fixed in a later change though.